docs/gha-workflows.md

```diff
@@ -145,3 +146,72 @@ jobs:
     with:
       github-run-id: "${{ github.event.workflow_run.id }}"
 ```
+# GitHub Actions: Integration Test Template Configuration
+
+This section details the inputs for the GitHub Actions workflow defined in `.github/workflows/test-integration-template.yaml` within the Camunda Platform Helm repository. These inputs allow you to customize Helm chart deployments for integration testing.
+
+## Workflow Inputs
```

Comment:
Please add a link to this section inside the workflow itself so it's easy to find.
https://github.com/camunda/camunda-platform-helm/blob/6ab109cc68bf5befd225be792cab29598da23dbf/.github/workflows/test-integration-template.yaml#L1

Reply:
I just fixed the name and directly linked it on there.